admin/qtx_admin_options.php

```diff
@@ -91,3 +91,39 @@ function qtranxf_admin_load_config() {
 
     qtranxf_add_conf_filters();
 }
+
+/**
+ * Import legacy option by recopying its value if the new option doesn't already exist.
+ *
+ * @param string $old_name
+ * @param string $new_name
+ * @param bool|string $autoload as in update_option
+ *
+ * @return void
+ */
+function qtranxf_import_legacy_option( $old_name, $new_name, $autoload = null ) {
+    assert( strpos( $new_name, 'qtranslate_' ) === 0 );
+    if ( ! get_option( $new_name ) ) {
```

Comment:
Not sure new named option should always have the priority. 
Maybe we could test also module activation state (test above `AND` module is active) though maybe involves a too remote scenario to complicate the function...

Reply:
The idea of the import is a one-shot action at activation, the goal is that QT-XT cares of its own options (modules).

The normal scenario is that new options don't exist yet. If we can import a value such as ACF or slugs legacy settings (coming from original plugins) the new option is created from these values to ensure the transition from the old plugins. But from that point we want to remain with the QT-XT setup and ignore the legacy stuff once we have all set in QT-XT.

The most tricky case is when the QT-XT options are reset, we don't really want to import again but in that case we expect the admin to save a new set of options right after the reset. If this is not done, a new import can happen again.

This patch is only valid for the modules as it's a new feature in QT-XT but this will be expanded to all other options later once we cut completely the links with QT-X (not XT).